backend/index.js

```diff
@@ -57,4 +58,37 @@ app.get('/env', (req, res) => res.status(200).send(`ENV: ${process.env.ENV}`));
 
 app.get('*', (req, res) => res.sendFile(path.join(process.cwd(), FE_BUILD_PATH, 'index.html')));
 
+//Creating a user with Prisma
+{
+  const prisma = new PrismaClient();
+
+  const createUser = async (name, email, password) => {
+    if (password.length < 8) {
+      throw new Error(`${password}: Password must be at least 8 characters long`);
+    };
+
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+    if (!emailRegex.test(email)) {
+      throw new Error(`${email}: Not valid email`);
+    };
+
+    const user = await prisma.users.create({
+      data: {
+        name,
+        email,
+        password
+      },
+    });
+    console.log(`Created user with ID: ${user.id}`);
+  };
+
+  try {
+    await createUser('John Doe', 'john.doe@example.com', 'password111');
+  } catch (err) {
+    console.log(err);
+  }
+
+};
```

Comment:
**suggestion (blocking):** Now create a separate file backend/routes/user.js, write this code snippet as a separate method, here u can check how to [create router and endpoint for it](https://www.codespeedy.com/create-separate-routes-file-in-node-express-js/), create a router for /user in backend/routes/user.js, where you can have this method on / post method. Get name, email, password from req.body object. Add user router in index.js . In case of question write @GagMirz . :D